Doc/whatsnew/3.2.rst

```diff
@@ -1060,15 +1060,15 @@ of nearly equal quantities:
 
 The :func:`~math.erf` function computes a probability integral or `Gaussian
 error function <https://en.wikipedia.org/wiki/Error_function>`_.  The
-complementary error function, :func:`~math.erfc`, is ``1 - erf(x)``:
+complementary error function, :func:`~math.erfc`, is ``1 - erf(x)``::
```

Comment:
This is not terribly important, but it would be nice to be more explicit and use something like

```rst
.. doctest::
   :options: +SKIP
```

to skip a doctest.